packages/editor/src/components/post-url/index.js

```diff
@@ -62,8 +61,11 @@ export default function PostURL( { onClose } ) {
 	const { editPost } = useDispatch( editorStore );
 	const { createNotice } = useDispatch( noticesStore );
 	const [ forceEmptyField, setForceEmptyField ] = useState( false );
-	const postUrlLabel = usePostURLLabel();
-	const copyButtonRef = useCopyToClipboard( postUrlLabel, () => {
+	const permalink = useSelect(
+		( select ) => select( editorStore ).getPermalink(),
+		[]
+	);
+	const copyButtonRef = useCopyToClipboard( permalink, () => {
```

Comment:
We should also use `safeDecodeURIComponent` for the copied permalink.

Reply:
@ntsekouras: Is it needed in practice? I had a quick look at `getPermalink`, `getPermalinkParts` and I don't see where we'd get encoded components from. Also:

<img width="932" alt="Screenshot 2024-05-23 at 10 24 02" src="https://github.com/WordPress/gutenberg/assets/150562/9399fd55-5729-406e-ba7e-8b9d30b3f75f">

In the above scenario I see `http://localhost:8881/2024/05/ola-φίλε/` in my clipboard, as expected.

Finally, I also just refactored to use the existing `useSelect` call in that component.

Reply:
Hm.. yesterday I tested with some greek and accents and it produced a URL that needed it. Don't remember what I tried exactly though and couldn't reproduce today with random tests, that could be similar to the yesterday ones..

Reply:
So I should just add the decoder to be safe?